### PR TITLE
NO-JIRA: Sync OSGi broker config with latest changes from ARTEMIS-1628

### PR DESCRIPTION
**Pull request by graben**

### CODE REVIEW

File: artemis-features/src/main/resources/artemis.xml [hunk 1]
```diff
@@ -32,21 +32,23 @@ under the License.
            MAPPED: mmap files
            NIO: Plain Java Files
        -->
-      <journal-type>ASYNCIO</journal-type>
+      <journal-type>NIO</journal-type>
```
Comment (michaelandrepearce) [inline artemis-features/src/main/resources/artemis.xml]: ASYNCIO should be the default it’s more performant and Artemis will fall back to NIO if not present

File: artemis-features/src/main/resources/artemis.xml [hunk 2]
```diff
@@ -32,21 +32,23 @@ under the License.
...
 
-      <paging-directory>./data/artemis/paging</paging-directory>
+      <paging-directory>./paging</paging-directory>
```
Comment (michaelandrepearce) [inline artemis-features/src/main/resources/artemis.xml]: Why change this?

File: artemis-features/src/main/resources/artemis.xml [hunk 3]
```diff
@@ -196,5 +194,20 @@ under the License.
 
       </addresses>
 
+
+      <!-- Uncomment the following if you want to use the Standard LoggingActiveMQServerPlugin pluging to log in events
```
Comment (michaelandrepearce) [inline artemis-features/src/main/resources/artemis.xml]: Why add this?

File: artemis-features/src/main/resources/artemis.xml (hunk 1 above)
Comment (graben) [reply artemis-features/src/main/resources/artemis.xml]: Well, I synced with a created default broker.xml its default is actually NIO and avoid "urgly" comments on every broker start if asyncio is not supported

File: artemis-features/src/main/resources/artemis.xml (hunk 2 above)
Comment (graben) [reply artemis-features/src/main/resources/artemis.xml]: root directory is data/artemis/<brokername>. This avoid data/artemis to be used unnecessary twice.

File: artemis-features/src/main/resources/artemis.xml (hunk 3 above)
Comment (graben) [reply artemis-features/src/main/resources/artemis.xml]: As mentioned before it's coming from default.

**Comment by jbertram** [comment]
Is there no option to just copy (or build) this file from the source so manual syncing isn't necessary?

**Comment by graben** [comment]
Well, actually the broker.xml is build from several templates and snippets. OSGi needs a complete file to trigger broker deployment. Therefore a a file need to be constructed during build process. Meight be tricky.

**Comment by graben** [comment]
Any further objection to not merge this?

**Pull request closed without merging**